Signal/src/ViewControllers/ExperienceUpgradesPageViewController.swift

```diff
@@ -4,6 +4,94 @@
 
 import Foundation
 
+private class IntroductingProfilesExperienceUpgradeViewController: ExperienceUpgradeViewController {
+
+    override func loadView() {
+        self.view = UIView()
+
+        /// Create Views
+
+        // Title label
+        let titleLabel = UILabel()
+        view.addSubview(titleLabel)
+        titleLabel.text = header
+        titleLabel.textAlignment = .center
+        titleLabel.font = UIFont.ows_regularFont(withSize: ScaleFromIPhone5(24))
+        titleLabel.textColor = UIColor.white
+        titleLabel.minimumScaleFactor = 0.5
+        titleLabel.adjustsFontSizeToFitWidth = true
```

Comment:
Can you please attach screenshots on plus size device?

Reply:
<img width="598" alt="screen shot 2017-09-05 at 4 28 28 pm" src="https://user-images.githubusercontent.com/217057/30090893-5d4b546c-9284-11e7-9757-8efebd795379.png">